.storybook/main.js

```diff
@@ -35,4 +35,8 @@ module.exports = {
     }
     return config;
   },
+  babel: async (options) => ({
+    ...options,
+    plugins: [['@babel/plugin-proposal-class-properties', { loose: true }]],
```

Comment:
It the issue you linked, they said in 6.3 it was fixed.. Sure you still had problems?

Reply:
I was still having problems but I'll double check

Reply:
Tried removing and deleted node modules and cache, and ran `yarn install --force`. Storybook failed to start with this error:
```
SyntaxError: \iTwinUI-react\src\core\Toast\Toaster.tsx: Missing class properties transform.
  17 |
  18 | export default class Toaster {
> 19 |   private toasts: ToastProps[] = [];
```

Reply:
hmm...well ok.
If we gonna wait for patch, lets try this again after updating.
Im gonna leave this open, unless we decide to merge.